contracts/teller-network/Escrow.sol

```diff
@@ -161,55 +157,29 @@ contract Escrow is Pausable, MessageSigned, Fees, Arbitrable, RelayRecipient {
      *         The expiration time must be at least 10min in the future
      *         For eth transfer, _amount must be equals to msg.value, for token transfer, requires an allowance and transfer valid for _amount
      */
-    function fund(uint _escrowId, uint _tokenAmount, uint _expirationTime) external payable whenNotPaused {
-        _fund(get_sender(), _escrowId, _tokenAmount, _expirationTime);
+    function fund(uint _tokenAmount, uint _expirationTime) external payable {
+        _fund(msg.sender, _tokenAmount, _expirationTime);
     }
 
-    function _fund(address _from, uint _escrowId, uint _tokenAmount, uint _expirationTime) internal whenNotPaused {
+    function _fund(address _from, uint _tokenAmount, uint _expirationTime) internal {
+        // TODO: expiration time
```

Comment:
What's left to do with the expiration?

Reply:
Currently it's handled as an input parameter. I think we should change this so the expiration time is calculated by the contract instead.